framework/src/application/network/network.ts

```diff
@@ -90,89 +91,87 @@ interface P2PRequest {
 }
 
 export class Network {
-	private readonly options: NetworkConfig;
-	private readonly channel: InMemoryChannel;
-	private readonly logger: Logger;
-	private readonly storage: Storage;
-	private secret: number | null;
-	private p2p!: liskP2p.P2P;
+	private readonly _options: NetworkConfig;
+	private readonly _channel: InMemoryChannel;
+	private readonly _logger: Logger;
+	private readonly _networkDB: KVStore;
+	private _secret: number | null;
+	private _p2p!: liskP2P.P2P;
 
 	public constructor({
 		options,
 		channel,
 		logger,
-		storage,
+		networkDB,
 	}: NetworkConstructor) {
-		this.options = options;
-		this.channel = channel;
-		this.logger = logger;
-		this.storage = storage;
-		this.secret = null;
+		this._options = options;
+		this._channel = channel;
+		this._logger = logger;
+		this._networkDB = networkDB;
+		this._secret = null;
 	}
 
 	public async bootstrap(): Promise<void> {
 		// Load peers from the database that were tried or connected the last time node was running
-		const previousPeersStr = await this.storage.entities.NetworkInfo.getKey(
+		const previousPeersStr = await this._networkDB.get<string>(
```

Comment:
This will fail with fresh database, so this should be in the `try-catch` and if it's `NotFoundError` then return `undefined`